micrometer-test-aspectj-ctw/build.gradle

```diff
@@ -6,9 +6,9 @@ plugins {
 description 'AspectJ compile-time weaving test for Micrometer aspects'
 
 dependencies {
-    implementation project(':micrometer-core')
     aspect project(':micrometer-core')
     aspect project(':micrometer-observation')
+    implementation libs.aspectjrt
```

Comment:
Without this, I see the following failure:

```
Execution failed for task ':micrometer-test-aspectj-ctw:compileJava'.
> Cannot infer AspectJ class path because no AspectJ Jar was found on class path: configuration ':micrometer-test-aspectj-ctw:runtimeClasspath'
```